slurp.1.scd

```diff
@@ -14,8 +14,8 @@ slurp is a command-line utility to select a region from Wayland compositors
 which support the layer-shell protocol. It lets the user hold the pointer to
 select, or click to cancel the selection.
 
-If the standard input is not a TTY, slurp will read a list of predefined
-rectangles for quick selection. Each line must be in the form
+If the standard input is not a TTY or the -r option is used, slurp will read a
```

Comment:
Unrelated to this PR: maybe we should have an option to force slurp to read from stdin, even if stdin is a TTY.

Reply:
That's probably a good idea. Maybe I'll make a seperate pr for that.

Reply:
Opened #67 for this.